meta/main.yml

```diff
@@ -26,9 +26,10 @@ galaxy_info:
         - buster
 
   galaxy_tags:
-    - raspberrypi
-    - raspbian
+    - smart
+    - assistant
     - hassio
+    - raspberry
```

Comment:
are these options specific to galaxy? If so, is there a list of these somewhere so you know which ones are "right"?